package.json

```
@@ -60,7 +60,7 @@
     "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "@capacitor/core": "^3.0.0 || ^4.0.0"
+    "@capacitor/core": "5.0.0" 
```

I would suggest to stick to:

```suggestion
    "@capacitor/core": "^5.0.0" 
```

Otherwise we would get peer dependency warnings as soon as the next patch version is released. This would also be in line with the dev dependencies.